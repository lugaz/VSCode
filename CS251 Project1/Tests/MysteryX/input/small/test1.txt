getRear
insertFront 76
deleteRear
insertRear 52
getFront
insertFront 586
deleteRear
insertFront 175
deleteRear
deleteFront
deleteFront
insertFront 692
deleteRear
insertFront 659
getFront
getFront
insertRear 346
deleteRear
insertFront 347
deleteRear
insertFront 673
deleteFront
insertFront 660
deleteFront
insertRear 276
deleteRear
