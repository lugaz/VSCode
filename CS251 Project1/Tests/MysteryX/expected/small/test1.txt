-1
76
52
52
586
175
-1
692
659
659
346
659
673
660
276
